spec/CordovaCheck.spec.js

```diff
@@ -47,27 +47,27 @@ describe('findProjectRoot method', function () {
         var somedir = path.join(home, 'somedir');
         var anotherdir = path.join(somedir, 'anotherdir');
         removeDir(somedir);
-        shell.mkdir('-p', anotherdir);
-        shell.mkdir('-p', path.join(somedir, 'www', 'config.xml'));
+        fs.ensureDirSync(anotherdir);
+        fs.ensureDirSync(path.join(somedir, 'www', 'config.xml'));
```

Comment:
I think the previous version was just a lazy hack. We actually want `fs.ensureFileSync` on all instances of this. One might might even consider  adding a test that asserts that directories are *not* accepted as `config.xml`.